src/App/Pages/Settings/SettingsPage/SettingsPageViewModel.cs

```diff
@@ -371,5 +407,15 @@ public void BuildList()
                 new SettingsPageListGroup(otherItems, AppResources.Other, doUpper)
             });
         }
+
+        private string getVaultTimeoutActionFromKey(string key)
+        {
+            return _vaultTimeoutActions.FirstOrDefault(o => o.Key == key).Value;
+        }
+
+        private int? getVaultTimeoutFromKey(string key)
+        {
+            return _vaultTimeouts.FirstOrDefault(o => o.Key == key).Value;
+        }
```

Comment:
Should be PascalCased

Reply:
....sorry..... pure muscle memory. Updating.